packages/host/app/components/matrix/room.gts

```diff
@@ -144,6 +149,11 @@ export default class Room extends Component<Signature> {
                 @submode={{this.operatorModeStateService.state.submode}}
                 @autoAttachedFile={{this.autoAttachedFile}}
                 @filesToAttach={{this.filesToAttach}}
+                @autoAttachedCardTooltipMessage={{if
```

Comment:
It would be clearer if both strings, "Current card is shared automatically", and "Topmost card is shared automatically" would be in the same logical construct, so that you don't have to check two different components to see which text would be rendered. For example:

`@autoAttachedCardTooltipMessage={{if this.isCodeMode 'Current card is shared automatically' 'Topmost card is shared automatically'}}`